addons/mail/static/tests/helpers/mock_models.js

```diff
@@ -129,6 +129,7 @@ export class MockModels {
                     needaction_partner_ids: { string: "Partners with Need Action", type: 'many2many', relation: 'res.partner' },
                     notification_ids: { string: "Notifications", type: 'one2many', relation: 'mail.notification' },
                     partner_ids: { string: "Recipients", type: 'many2many', relation: 'res.partner' },
+                    recipients: { string: "Recipients", type: 'many2many', relation: 'res.partner' },
```

Comment:
This is not a field in python so no need to add it, what needs to be changed is the result of message format.